docs/guides/zone-replication.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Ingester Hand-over"
+linkTitle: "Ingester Hand-over"
+weight: 5
+slug: ingester-handover
```

Comment:
@khaines This comes from a bad copy-paste. May you submit a PR to fix it, please?

Reply:
yes it did... looked at it a few times and this didn't catch my eye.